src/central.ts

```diff
@@ -80,4 +80,29 @@ export class Central implements Ble {
     )
     return onDiscoverPeripheralListener
   }
+
+  registerOnConnectedListener(
+    cb: ({
+      peripheralId,
+      name,
+    }: {
+      peripheralId: string
+      name?: string
+    }) => void
+  ) {
+    const bleDidcommEmitter = new NativeEventEmitter(NativeModules.BleDidcomm)
```

Comment:
this should be a property on the class. It can be added to both the central and peripheral OR the Ble interface can be an abstract class with the property defined there. I am fine with either.